src/control/JacobianVelController.h

```diff
@@ -4,6 +4,7 @@
 #include "../filters/StateSpaceUtil.h"
 #include "../navtypes.h"
 #include "../utils/math.h"
+#include "../world_interface/data.h"
```

Comment:
When I was including the JacobianVelController.h in my base motor class, it required that I include data.h before JacobianVelController.h in order for my code to build properly. However, the formatting check was failing because it wanted data.h to be at the end of my include statements, which would cause the build check to fail, so this was my workaround to get all of the checks to pass.